hack/jenkins/benchmark-dockerized.sh

```diff
@@ -34,6 +34,9 @@ retry() {
 export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
 
 retry go get github.com/cespare/prettybench
+if [[ -n "${KUBE_JUNIT_REPORT_DIR:-}" ]]; then
+  go get github.com/jstemmer/go-junit-report
```

Comment:
Is this (and prettybench) vendored? If not they should be.

We should not be downloading code (flaky and/or disapper) during test execution.

Reply:
I agree with @fejta. Also not sure testing on this is right.. we either call go-junit-report or we don't.. I don't think that hinges on if we have the report dir set, does it? If it does, can you link to that?

Reply:
About vendoring prettybench, I agree with @fejta.
However, I would like to focus on fixing hack/jenkins/benchmark-dockerized.sh first.
I can file an issue about downloading golang modules instead of vendoring it. For example `go-junit-report` is also downloaded in hack/jenkins/gotest.sh and hack/jenkins/test-dockerized.sh.

Reply:
If `benchmark-dockerized.sh` needs `go-junit-report`, it should `go install` the vendored version like our other utilities do: https://github.com/kubernetes/kubernetes/blob/27d495840954ec386509e210beb240cb77c13ce7/hack/update-bazel.sh#L28

As long as we're fixing this, we should also fix prettybench to be a go install command rather than go get, but feel free to punt on that if you'd prefer.

Reply:
I created separate PR #73740 to add go-junit-report and prettybench to the vendor.

Reply:
Thanks! Gave that one a review.

Reply:
Thanks :)